src/server/api/routes/users.router.js

```diff
@@ -62,6 +62,40 @@ router.get('/:id', (req, res, next) => {
     .then((result) => res.json(result))
     .catch(next);
 });
+/**
+ * @swagger
+ * /users/{user_id}/favorites:
+ *  get:
+ *    tags:
+ *    - Favorites
```

Comment:
This should be under Users routes

Reply:
Yes thanks, actually i created separate routers and controllers files for favorites before, but later i just put the favorites part into users. So i forgot to change it to Users :) Thanks